handbook/engineering/hiring/engineering-manager-web.md

```diff
@@ -42,25 +44,25 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
        - If we have at least one of the above, the second interviewer can be [any engineer on the web team](../web/index.md#members).
    - 1h [Engineering leadership](engineering-leadership.md).
      - [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) (required)
-     - Plus one of:
+     - Plus one of the following:
          - [Tomás Senart](../../../company/team/index.md#tomás-senart)
          - [Loïc Guychard](../../../company/team/index.md#loïc-guychard)
          - [Aida DeWitt](../../../company/team/index.md#aida-dewitt)
          - Can backfill with any [engineering manager](../leadership/index.md#members).
-   - 1h [Technical experience](https://github.com/sourcegraph/interviews/blob/master/engineering/technical-experience.md)
-     - Pick at least one (ideally both) of:
+   - 1h Technical experience - We ask you about your past work and accomplishments.
```

Comment:
Still valuable to keep the link for for interviewers, just need to mark it as such. Team collaboration interview needs same treatment.

```suggestion
   - 1h Technical experience - We ask you about your past work and accomplishments. [Internal documentation for interviewers](https://github.com/sourcegraph/interviews/blob/master/engineering/technical-experience.md).
```